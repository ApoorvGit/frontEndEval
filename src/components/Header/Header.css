@import url('https://fonts.googleapis.com/css2?family=Big+Shoulders+Inline+Display:wght@700&family=Raleway:wght@100&display=swap');
.header {
  height: 105px;
  background-color: black;
  width: 100%;
}
.header > h1 {
  color: white;
  padding-left: 10%;
  padding-top: 20px;
  font-family: 'Big Shoulders Inline Display', cursive;
  font-size: 62px;
  font-weight: 500;
}
