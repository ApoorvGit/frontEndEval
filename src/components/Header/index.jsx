import React from 'react';
import './Header.css';

export default function Header() {
  return (
    <div className="header">
      <h1>Eventify</h1>
    </div>
  );
}
