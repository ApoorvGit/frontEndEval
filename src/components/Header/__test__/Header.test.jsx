import { render } from '@testing-library/react';
import React from 'react';
import Header from '../index';
import '@testing-library/jest-dom';

describe('Header', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
