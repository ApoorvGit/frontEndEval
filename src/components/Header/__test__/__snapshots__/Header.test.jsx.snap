// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <h1>
      Eventify
    </h1>
  </div>
</DocumentFragment>
`;
